server/gokbg3/grails-app/domain/org/gokb/cred/Package.groovy

```diff
@@ -287,6 +287,16 @@ class Package extends KBComponent {
     result
   }
 
+  @Transient
+  public getTippCount() {
+    def combo_tipps = RefdataCategory.lookup(RCConstants.COMBO_TYPE, 'Package.Tipps')
+
+    int result = Combo.executeQuery("select count(c.id) from Combo as c where c.fromComponent = ? and c.type = ?"
```

Comment:
using ? as argument is deprecated and against convention

Reply:
Legacy

Reply:
done in separate process